VikingQuiz.Api/Controllers/EmailController.cs

```diff
@@ -54,11 +57,10 @@ public IActionResult SendEmail(int id)
                 return NotFound();
             }
 
-            // we want the 'confirm email' token to be valid for 168 / 24 = 7 days
-            // this token should only be usable to reset password.
-            string token = authenticationService.GenerateTokenForUser(user, 168, "email");
             string Subject = "Register your account";
-            string Body = "Click the following link to register your account\n\nhttp://localhost:3000/confirm/" + token;
+            string token = authenticationService.GenerateTokenForUser(user, ConfirmEmailExpiryHours, "email");
+            string registerAccountAddress = "http://localhost:3000/confirm/" + token;
```

Comment:
Why isn`t the url extracted in a constant and the endpoint built using concatenation?